packages/components/src/box/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { color, space, layout, flexbox, background, border, position, shadow }
+	from 'styled-system';
+
+const Box = styled.div`
```

Comment:
Does this automatically translate to `View` on native or would we need a separate implementation?

Reply:
no, we'll need a sparate box/index.native.js for that. Planning to open a separate PR for mobile

Reply:
Oh, I was hoping styled-components would bring more reusability there. I imagine the implementation of that would be exactly the same except using `styled.View` instead of `styled.div`?

How would we implement other components, like the one to replace the `<figure>` in `GalleryImage`? Would that also be the same as this but a `styled.figure`?

Borrowing from the previous `HTMLElement` experiment, I would expect `Box` to be a more generic container thing that would accept a `tagName` prop to be used on the web implementation. 

I think the syntax of styled-components would allow for this, but I'm not sure what that looks like yet

Reply:
> I imagine the implementation of that would be exactly the same except using styled.View instead of styled.div?

Right, that's also an option. We can do a platform check and call `styled.View` if it is mobile.

> How would we implement other components, like the one to replace the <figure> in GalleryImage? Would that also be the same as this but a styled.figure?

yes, you can either use a valid react/react-native component or a tagname like div, figure, h1 etc.